packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -552,7 +552,7 @@ export class ScheduleManager {
             this.deltaManager,
             ChildLogger.create(this.logger, "DeltaScheduler"),
         );
-        this.scheduler = new ScheduleManagerCore(deltaManager, logger);
+        void new ScheduleManagerCore(deltaManager, logger);
```

Comment:
Future todo: ScheduleManager/ScheduleManagerCore should implement IDisposable and clean up appropriately (unregister events, etc.) when the ContainerRuntime disposes, esp. since it has the power to pause/resume the DeltaManager.